CONTRIBUTING.md

```diff
@@ -32,6 +32,10 @@ Also, make sure that your PRs do not contain unnecessary commits. If you think
 some of your commits should be merged into a single one, feel free to [squash
 them](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
 
+Please [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) outdated
+PRs on master to help with the reviews (rebasing is preferred over merging to
+keep a clean history in a branch/PR).
```

Comment:
You don't think adding a quick command line would help? `git rebase master` or something like this?
Oh wait, folks also need to update their fork wrt the upstream repo...